cmds/task/task.go

```diff
@@ -38,6 +55,14 @@ func init() {
 
 	retriggerCmd.Flags().BoolP("exact", "e", false, "Retrigger in exact mode. WARNING: THIS MAY HAVE SIDE EFFECTS. USE AFTER YOU READ THE SOURCE CODE.")
 
+	rerunCmd.Flags().BoolP("noop","n", false, "Specifies the operation to perform and adjust some operations to print extra information.")
```

Comment:
Language nit: `--noop` does not specify the operation to perform. It tells the command to not actually run the command, but print out what it would do